11/Task11/Class1.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task11
+{
+    public class Tour
+    {
+        public string Name { get; }
+        public readonly int Code;
+        public int Dur;
+        public List<string> Plan;
+        public TransportType Transport;
+        public double Price { get; }
+        public DateTime Start;
+        public string Description;
+        public DateTime End 
+        { get
+            {
+                return Start.AddDays(Dur);
+            } 
+        }
+        public Tour(string name, int code, List<string> plan, TransportType transport, int duration)

```

Comment:
И в конструкторе параметр, соответствующий продолжительности, лучше называть так, чтобы для пользователя (программиста, который будет пользоваться вашим классом и, в частности, конструктором) были понятны единицы измерения этой величины
```suggestion
        public Tour(string name, int code, List<string> plan, TransportType transport, int durationInDays)

```